src/service/platform-impl.js

```diff
@@ -136,6 +136,25 @@ export class Platform {
     }
     return parseInt(res[index], 10);
   }
+
+  /**
+   * Returns the minor ios version in string.
+   * The ios version can contain two numbers (10.2) or three numbers (10.2.1).
+   * Direct string equality check is not suggested, use startWith instead.
+   * @returns {string}
+   */
+  getIosVersion() {
```

Comment:
`getMajorVersion` is for browser major version. I thought if we have a `getMinorVersion`, that should be for browser too, but there's no such use case yet. Also, there's no use case for getting android version, so that's why I only created `getIosVersion`